r/datasync_location_fsx_windows - add resource (#12686)
terraform-providers/terraform-provider-aws: website/docs/r/datasync_location_fsx_windows_file_system.html.markdown

---
subcategory: "DataSync"
layout: "aws"
page_title: "AWS: aws_datasync_location_fsx_windows_file_system"
description: |-
  Manages an FSx Windows Location within AWS DataSync.
---

# Resource: aws_datasync_location_fsx_windows_file_system

Manages an AWS DataSync FSx Windows Location.

## Example Usage

```hcl

> resource "aws_datasync_location_fsx_windows_file_system" "example" {

Reviewer: Terraform 0.12 syntax now for examples I think after #8950.

Author: oops, i though CI already covers this. fixing.

  fsx_filesystem_arn  = "${aws_fsx_windows_file_system.example.arn}"
  user                = "SomeUser"
  password            = "SuperSecretPassw0rd"
  security_group_arns = ["${aws_security_group.test1.arn}"]
}
```

## Argument Reference

The following arguments are supported:

* `fsx_filesystem_arn` - (Required) The Amazon Resource Name (ARN) for the FSx for Windows file system.
* `password` - (Required) The password of the user who has the permissions to access files and folders in the FSx for Windows file system.
* `user` - (Required) The user who has the permissions to access files and folders in the FSx for Windows file system.
* `domain` - (Optional) The name of the Windows domain that the FSx for Windows server belongs to.

> * `security_group_arns` - (Optional) The Amazon Resource Names (ARNs) of the security groups that are to use to configure the FSx for Windows file system.

Reviewer: ```suggestion
* `security_group_arns` - (Optional) The Amazon Resource Names (ARNs) of the security groups used to configure the FSx for Windows file system.
```
